src/core/js/adapt.js

```diff
@@ -398,8 +402,14 @@ define([
     async remove() {
       const currentView = this.parentView;
       if (currentView) {
-        currentView.model.setOnChildren('_isReady', false);
-        currentView.model.set('_isReady', false);
+        currentView.model.setOnChildren({
+          '_isReady': false,
+          '_isRendered': false
+        });
+        currentView.model.set({
+          '_isReady': false,
+          '_isRendered': false
+        });
```

Comment:
I don't think this is needed as `setOnChildren` applies the settings to the model **and** its children

Reply:
that's true